server/lib/config/config.ts

```diff
@@ -45,6 +48,28 @@ function assert(assertion: Boolean, msg: string)
 	if (!assertion)
 		throw new Error(msg);
 }
+// add automatic IP detection
+function getListenIps() {
```

Comment:
If the network interface name doesn't match to the whitelist regex pattern, I'm kicking out of the a room when trying to join. This function must provide a fallback to "0:0:0:0 " when the interface was not matched.

My case on debian 10: interface called "wlp2s0" (it stands for "wlan") accroding to the convention described for example here: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-understanding_the_predictable_network_interface_device_names

Reply:
Added fallback functionality and expanded interface whitelist according naming convention. Please review it again. 
I will try to find if there is more elegant solution for automatic IP detection without need to have hard coded whitelist.